### ISSUE TITLE
Crash if depfile adds dependency between several outputs of an edge

### ISSUE THREAD

**Issue by nico** [ninja-build/ninja#875]
```
$ cat build.ninja
rule cgt
   depfile = c.d
   command = touch a; touch b; echo "a: b" > c.d

build a b: cgt

$ ninja && ninja
Segmentation fault: 11
```

**Comment by halton** [comment]
Which version I can get this fix? My ninja is 1.5.3 (from depot_tools), still get the segment fault.

**Comment by nico** [comment]
There hasn't been a release with this fix yet. I want to make a new release in the not-too-distant future though.
